sound/soc/sof/intel/icl.c

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Authors: Fred Oh <fred.oh@linux.intel.com>
+//
+
+/*
+ * Hardware interface for audio DSP on IceLake.
+ */
+
+#include "../ops.h"
+#include "hda.h"
+#include "hda-ipc.h"
+#include "../sof-audio.h"
```

Comment:
you need more headers for `ID_ENABLED()`, `CONFIG_*`, `EXPORT_SYMBOL_NS()`, `GENMASK()`, `ARRAY_SIZE()` etc.